glyphs-reader/src/font.rs

```diff
@@ -1206,6 +1100,43 @@ impl Font {
     }
 }
 
+/// Convert [kurbo::Point] to this for eq and hash/
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+struct PointForEqAndHash {
+    x: OrderedFloat<f64>,
+    y: OrderedFloat<f64>,
+}
+
+impl PointForEqAndHash {
+    fn new(point: Point) -> PointForEqAndHash {
+        point.into()
+    }
+}
+
+impl From<Point> for PointForEqAndHash {
+    fn from(value: Point) -> Self {
+        PointForEqAndHash {
+            x: value.x.into(),
+            y: value.y.into(),
+        }
+    }
+}
+
+/// Convert [kurbo::Affine] to this for eq and hash/
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+struct AffineForEqAndHash([OrderedFloat<f64>; 6]);
+
+impl From<Affine> for AffineForEqAndHash {
+    fn from(value: Affine) -> Self {
+        let coeffs = value.as_coeffs();
+        let mut ordered_float_coeffs = [OrderedFloat::<f64>::from(0.0); 6];
+        for i in 0..6 {
+            ordered_float_coeffs[i] = coeffs[i].into();
+        }
+        AffineForEqAndHash(ordered_float_coeffs)
```

Comment:
```suggestion
        Self(value.as_coeffs().map(|coeff| coeff.into()))
```

Thanks to const generics, [`map`](https://doc.rust-lang.org/std/primitive.array.html#method.map) now does wonderful things on arrays.